python/latticex/rosetta/__init__.py

```diff
@@ -38,6 +38,10 @@ def __getattr__(self, attr):
 sys.stdout = ZeroBufferOut(sys.stdout)
 
 
+if 'ROSETTA_MPC_128' in os.environ and os.environ['ROSETTA_MPC_128'] == 'ON':
```

Comment:
I see the same codes in multi files, could you make them in a separate file and import it in other files?

Reply:
I have push request to fix the issue: [https://github.com/LatticeX-Foundation/Rosetta/pull/31/commits/485acf4a267f633d581d9a27edd9e3abd81a9117](update)